discovery_engine/lib/src/domain/repository/type_id.dart

```diff
@@ -21,3 +21,4 @@ const newsResourceTypeId = 5;
 const uriTypeId = 6;
 const durationTypeId = 7;
 const stackIdTypeId = 8;
+const embeddingAdapter = 9;
```

Comment:
to align it with the other type ids
```suggestion
const embeddingTypeId = 9;
```